src/psbt.cpp

```diff
@@ -123,13 +123,26 @@ CMutableTransaction PartiallySignedTransaction::GetUnsignedTx(bool force_unblind
         txin.nSequence = input.sequence.value_or(max_sequence);
         txin.assetIssuance.assetBlindingNonce = input.m_issuance_blinding_nonce;
         txin.assetIssuance.assetEntropy = input.m_issuance_asset_entropy;
-        if (input.m_issuance_value != std::nullopt && input.m_issuance_inflation_keys_amount != std::nullopt && force_unblinded) {
+        // If there is a commitment we should set the value to the commitment unless we are forcing unblinded.
```

Comment:
This code handled blinded vs. unblinded issuances badly. It always tried to blind the issuances, even if they were unblinded issuance assets.

Since there are no fields in the PSET spec to define if an issuance should be blinded or not, we will use the presence of the issuance blinding commitment to tell us what to do.

Reply:
This looks similar that it may solve the problem. The use case I was trying to solve was an unblinded re-issuance.

Reply:
Just tested #1119 which provides a testcase for #1103 and it is not fixed by this